dist/dist.go

```diff
@@ -0,0 +1,10 @@
+/*
+Distribution the cothority-program in binary form. At least the
+following two distributions should be available:
+
+	* .tar.gz
+	* docker
+
+Others may or may not follow.
+*/
+package dist
```

Comment:
Ach so was. I hesitate to delete it - it's kind of the future distribution of cothority. I deleted it before but put it back. If you think it's not worth it, I'll delete it again ;)